pages/api/content/index.js

```diff
@@ -1,5 +1,24 @@
 import sql from "~/lib/postgres";
 
-const handler = async (req, res) => res.status(200).json({ data: await sql`SELECT * FROM content;` });
+const handler = async (req, res) => {
+	if (req.method === "GET") {
+		return res.status(200).json({ data: await sql`SELECT * FROM content;` });
+	}
+	if (req.method === "POST") {
+		const { title, content_type, description, img_url, video_url } = req.body;
 
+		const checkContentTitle = await sql`SELECT * FROM content WHERE title=${title};`;
+
+		if (!title || !content_type || !description) {
+			return res.status(400).json({ message: "Please fill the required fields" });
+		}
+		if (checkContentTitle.count !== 0) {
```

Comment:
This may be part of the design - we're definitely not allowing repeated titles? If that's a decision that's been made, then this is fine. It just seems like it might be a bit restrictive, over time I'd imagine short titles would start to appear more than once.

Reply:
It wasn't discussed , my idea was not to have duplicate titles in the database